datalad/__init__.py

```diff
@@ -243,8 +244,8 @@ def teardown_package():
         print("Obscure filename: str=%s repr=%r"
                 % (OBSCURE_FILENAME.encode('utf-8'), OBSCURE_FILENAME))
     except UnicodeEncodeError as exc:
-        from .dochelpers import exc_str
-        print("Obscure filename failed to print: %s" % exc_str(exc))
+        ce = CapturedException(exc)
+        print("Obscure filename failed to print: %s" % ce)
```

Comment:
sorry if I forgot, why 
1. not 
```suggestion
        print("Obscure filename failed to print: %s" % CapturedException(exc))
```
2. and not just overload `exc_str = CapturedException` and not bother with any code changes whenever it is used? (AFAIK it is always used/expected to provide a str)